Issues/Week72.md

```diff
@@ -7,6 +7,9 @@
 * [Swift in Production: Scenery](http://chris.eidhof.nl/posts/scenery-launch.html), by [@chriseidhof](https://twitter.com/chriseidhof)
 * [Introduction to Custom View Controller Transitions and Animations](http://www.appcoda.com/custom-view-controller-transitions-tutorial/), by [@joyceechessa](https://twitter.com/joyceechessa)
 * [Artsy's Technology Stack, 2015](http://artsy.github.io/blog/2015/03/23/artsy-technology-stack-2015/), by [@dblockdotorg](https://twitter.com/dblockdotorg)
+* [Protocols and Generics](http://airspeedvelocity.net/2015/03/26/protocols-and-generics-2/), by [@AirspeedSwift](https://twitter.com/AirspeedSwift)
+* [Evaluating Expressions in iOS with Objective-C and Swift](http://spin.atomicobject.com/2015/03/24/evaluate-string-expressions-ios-objective-c-swift/), by Matt Nedrich  [@atomicobject](https://twitter.com/atomicobject)
+* [ReactiveCocoa Concepts for Asynchronous Libraries](http://spin.atomicobject.com/2015/03/19/reactivecocoa-asynchronous-libraries/), by Patrick Bacon [@atomicobject](https://twitter.com/atomicobject)
 
```

Comment:
If I didn't put was because I didn't found it...

On Thu, Mar 26, 2015 at 10:06 AM, Rui Peres <notifications@github.com>
wrote:

> In Issues/Week72.md
> <https://github.com/iOS-Goodies/iOS-Goodies/pull/48#discussion_r27200185>:
>
> > @@ -7,6 +7,9 @@
> >  * [Swift in Production: Scenery](http://chris.eidhof.nl/posts/scenery-launch.html), by [@chriseidhof](https://twitter.com/chriseidhof)
> >  * [Introduction to Custom View Controller Transitions and Animations](http://www.appcoda.com/custom-view-controller-transitions-tutorial/), by [@joyceechessa](https://twitter.com/joyceechessa)
> >  * [Artsy's Technology Stack, 2015](http://artsy.github.io/blog/2015/03/23/artsy-technology-stack-2015/), by [@dblockdotorg](https://twitter.com/dblockdotorg)
> > +* [Protocols and Generics](http://airspeedvelocity.net/2015/03/26/protocols-and-generics-2/), by [@AirspeedSwift](https://twitter.com/AirspeedSwift)
> > +* [Evaluating Expressions in iOS with Objective-C and Swift](http://spin.atomicobject.com/2015/03/24/evaluate-string-expressions-ios-objective-c-swift/), by Matt Nedrich  [@atomicobject](https://twitter.com/atomicobject)
> > +* [ReactiveCocoa Concepts for Asynchronous Libraries](http://spin.atomicobject.com/2015/03/19/reactivecocoa-asynchronous-libraries/), by Patrick Bacon [@atomicobject](https://twitter.com/atomicobject)
> >
>
> The twitter handlers for Atomicobject's blog posts should reference the
> author twitter handler.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iOS-Goodies/iOS-Goodies/pull/48/files#r27200185>.
>